source/common/http/conn_manager_impl.h

```diff
@@ -497,10 +497,9 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
 
     void traceRequest();
 
-    // Updates the snapped_route_config_ if scope found, or ends the stream by
-    // sending local reply.
-    // Returns true if scoped route config snapped, false otherwise.
-    bool snapScopedRouteConfig();
+    // Updates the snapped_route_config_, if scope not found for the header, a
```

Comment:
Updates the snapped_route_config_.  If a scope is not found, snapped_route_config_ is set to Route::NullConfigImpl.

Reply:
done.